src/utils/serum.ts

```diff
@@ -5,8 +5,14 @@ import type { ISerumMarketInfo } from "../types/serum";
 export const SERUM_MARKET_LIST_URL =
   "https://raw.githubusercontent.com/step-finance/serum-markets/main/src/markets.json" as const;
 
-export const getSerumMarketInfoMap = async (): Promise<ISerumMarketInfo[]> => {
-  const serumMarketResponse = await axios.get<ISerumMarketInfo[]>(SERUM_MARKET_LIST_URL);
-  const { data: serumMarketList } = serumMarketResponse;
-  return serumMarketList;
+export const getSerumMarketInfoMap = async (): Promise<ISerumMarketInfo[] | undefined> => {
+  try {
+    const serumMarketResponse = await axios.get<ISerumMarketInfo[]>(SERUM_MARKET_LIST_URL);
+    const { data: serumMarketList } = serumMarketResponse;
+    return serumMarketList;
+  } catch(err) {
+    console.log(err);
+    return;
```

Comment:
Why not just return an empty an array instead of dual return types?

Reply:
ah yeah good point. Just updated that and star atlas as well